java/src/main/resources/com/dke/data/agrirouter/proto/technicalmessagetype/gps.proto

```diff
@@ -36,8 +36,18 @@ message GPSList{
         uint32 number_of_satellites = 7;
         google.protobuf.Timestamp gps_utc_timestamp = 8;
         FieldStatus field_status = 9;//Are we on a road, in a field or just somewhere offroad?
+        uint32 source_device_index = 10;//The index of the source device in the SourceDeviceList
         repeated google.protobuf.Any extension = 2048;  // Used for proprietary purposes
     }
+
+    message SourceDevice{
+        uint32 index = 1;//The index within the list of SourceDevices; Not necessarilly all values are there
```

Comment:
```suggestion
        uint32 index = 1;//The index within the list of source devices; Not necessarily all values are there
```
I am not quite sure, what this means "Not necessarily all values are there", perhaps this has to be clearer? Does this mean, that the value is optional?

Reply:
Sascha's textual comment is not incorporated. I guess what you want to say is that there may be gaps in those indexes? 
Suggestion:
"Indexes are not guaranteed to be consecutive, or of any specific form."